lib/pkcs11/pkcs11_config.c

```diff
@@ -480,7 +480,7 @@ CK_RV pkcs11_config_key(pkcs11_lib_ctx_ptr pLibCtx, pkcs11_slot_ctx_ptr pSlot, p
         }
     }
 
-    if (i < 16)
```

Comment:
There are only 16 slots in a cryptoauth device which is what this value represents - there isn't a #define for number of slots in a device. Thus freeslots may not be greater than 16 - however if one were to configure all slots in an ECC device as private keys then you'd actually require 32 objects to represent them as the specification requires that every private key object to have a corresponding public key object.